README.md

```diff
@@ -1,2 +1,80 @@
-# beacon-starter
+# Beacon Reader Example
 
+> A starter project for reading beacon values from a smart contract
+
+This project is composed of the following:
+
+1. A test that uses a mocked RrpBeaconServer to simulate reading a beacon value from a smart contract.
+1. A script to deploy the BeaconReaderExample contract to any network. This script will set the address of the RrpBeaconServer contract deployed on the selected network and after the script finishes it will display the address of the contract deployed but also save this address to `deployments` directory on the root of this repo (except for hardhat network since this network is started and stop just for the duration of the script execution).
+1. A script to read the beacon value from the deployed contract. If selected network is not `hardhat` or `localhost` then this address must be then whitelisted by the RrpBeaconServer manager account prior to reading the beacon value.
+
+## Instructions
+
+1. Clone this repo
+1. Run the following to install the dependencies
+
+```sh
+npm install
+```
+
+Now depending on the network you are using you will need to do the following:
+
+### Default network
+
+Simplest way to test this project is by using the integrated [Hardhat network](https://hardhat.org/hardhat-network/) and not extra steps are needed in this case to run both the test and the scripts.
+
+#### Test
+
+```sh
+npx hardhat test test/BeaconReaderExample.test.js
+```
```

Comment:
I can get rid of the warning in `BeaconReaderExample` by just using the timestamp but I don't think is worth doing anything about the unused variable warning in the `RrpBeaconServerMock` contract since it's used only for local testing.

Reply:
Agreed- the one remaining warning isn't a big deal